packages/block-library/src/site-logo/edit.js

```diff
@@ -141,6 +141,10 @@ const SiteLogo = ( {
 	// becomes available.
 	const maxWidthBuffer = maxWidth * 2.5;
 
+	// Set the default width to a responsible size.
+	// Note that this width is also set in the attached CSS file.
+	const defaultWidth = 120;
```

Comment:
This is something that we should express in theme.json as a default as well cc @nosolosw

Reply:
Tracking it at https://github.com/WordPress/gutenberg/issues/31037